internal/metrics/alerts/collector.go

```diff
@@ -0,0 +1,75 @@
+package alerts
+
+const CollectorPrometheusAlert = `
+"groups":
+- "name": "logging_collector.alerts"
+  "rules":
+  - "alert": CollectorNodeDown
```

Comment:
@xperimental  Do you think need to create one of these per deployed namespace?  What would that mean for the recording rules.  I assume if we have one set of alerts/rules per CLF > 0 then we need these expressions to somehow take into account all namespaces or be restricted to the namespaces of all CLF